### ISSUE TITLE
Delete images from AU workstation computer

### ISSUE THREAD

**Issue by beckerah** [NHMDenmark/Herbarium-Sheets-workstation#154]
I need to delete the images that have been successfully ingested from the AU workstation. This relates to issue [#149](https://github.com/NHMDenmark/Herbarium-Sheets-workstation/issues/149). I've written a script to delete the images and tried to help Birgitte run it over zoom, but it didn't work out. Now we are trying to get remote access to that computer so hopefully I can get in that way and delete the images myself. 

Status of gaining remote access can be followed here: [#542](https://github.com/NHMDenmark/Mass-Digitizer/issues/542).

**Comment by beckerah** [comment]
I found ten folders on the AU workstation computer that had no ingested images in them, so Jesper and I helped Birgitte and Charlotte begin manually ingesting those. It's taking several days but they are working on it. This process has freed up enough space in the meantime for Charlotte to continue digitizing, however she is going to put the images in a different folder than usual, so Aksel's code doesn't run on them yet. This is because species-web is down, and species-ocr doesn't seem to be working correctly.

**Comment by beckerah** [comment]
Charlotte wants to know if she can delete the folder E:\DaSSCo_Digitization\20241014_AU_Herb02_Fabaceae_CVH from their workstation computer. She thinks they were ingested since there is no error log but 322 images are still in the folder. There were originally 506 images in this folder. They all have the same date_asset_taken of October 14, 2024. I'm running the barcode-guid matching script for the date she says they were ingested (Nov 12) and when it's finished I'll check to see how many images have this date_asset_taken. That should give us an idea if all the images from this folder were ingested or not.

**Comment by beckerah** [comment]
I deleted a lot of images and associated files today but I ran into a couple of issues:

1. Some of the tif images have a different name than the raw files. It's okay if the raw file names start with the same as the tif because the script can still find and delete these, but for others, they're harder to find. I'll have to figure out how to fix this. Maybe I will have to make a list of all files to keep and have it delete everything else in the folder.
2. Some of the image folders have been moved to a different drive. Birgitte is going to move them back and put them all in the same folder so I can access these and delete them as well.
3. It looks like there are additional raw files in some of the folders that were not on my list of images to delete. I'll have to figure out what the deal is for these and if they need to be exported to tifs as well.

However, they have at least 4 TB of space free now so Charlotte should be good to return to digitizing when she's back at AU next week.
